docker/db/initdb.d/setup.sql

```diff
@@ -1,6 +1,23 @@
-CREATE TABLE Users(
-  account_id        SERIAL PRIMARY KEY,
-  account_name      VARCHAR(20),
-  email             VARCHAR(100),
-  password    CHAR(64)
-);
\ No newline at end of file
+CREATE TABLE Shops(
+  shop_id        PRIMARY KEY,
+  shop_name      VARCHAR(40),
+);
+
+CREATE TABLE Shop_Time(
+  sales_day      VARCHAR(40),
```

Comment:
VARCHAR(40)の意図は
どんな型で格納する予定？
Sunday？，日？，日曜日？
```suggestion
  sales_day      VARCHAR(40), NOT NULL
```